### DISCUSSION TITLE
How to get individual results of each nodes?

### DISCUSSION THREAD

**Discussion by faghiogbaogbaogho** [HewlettPackard/swarm-learning#174] [General]
The framework can output the final merged model, but can I get the last individual model of each nodes?

**Comment by iArpanPatel** [comment]
Could you please elaborate on the use case?? 
Final Swarm model will be same on all the nodes. Even the global merged model at the end of each sync round, will be the same.

**Reply by iArpanPatel** [reply]
@faghiogbaogbaogho please check our response and respond back.
